src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// category check action
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: 
`const CHECK_STATUS = bookstore/categories/CHECK_STATUS`